HunterPie.Core/Game/World/Entities/MHWMonster.cs

```diff
@@ -241,6 +270,16 @@ private void GetMonsterEnrage()
             enrage.Update(enrageStructure);
         }
 
+        [ScannableMethod]
+        private void GetMonsterCaptureThreshold()
+        {
+            MonsterDataSchema? monsterSchema = MonsterData.GetMonsterData(Id);
+            if (monsterSchema is null)
+            {
+                return;
+            }
+            CaptureThreshold = monsterSchema.Value.Capture/100f;
+        }
```

Comment:
`ScannableMethod` is a method that is called every X milliseconds, the Capture threshold for MHW is hardcoded and since it's hardcoded, there's no need for it to be updated every time.

```suggestion
public float CaptureThreshold => MonsterData.GetMonsterData(Id)?.Capture / 100f ?? 0;
```

Reply:
Yeah it makes sense. But, when I do it this way, on the first time I load a level, it seems to receive monster ids with -1 values and I cannot get the capture data. 

A workaround (without using the ScannableMethod) I found is to use the same "hack" that we use for the monster weakness (MHR and MHW) where we get those values in the Id setter [here](https://github.com/Haato3o/HunterPie-v2/blob/c33982d33337fb573692fa41238dbc8391112922/HunterPie.Core/Game/Rise/Entities/MHRMonster.cs#L49). I committed those changes for now, but maybe there is a better way to fix this issue?